app/src/main/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeSheet.kt

```diff
@@ -1,15 +1,29 @@
 package org.simple.clinic.facility.change.confirm
 
+import android.content.Context
+import android.content.Intent
 import android.os.Bundle
 import org.simple.clinic.ClinicApp
 import org.simple.clinic.R
+import org.simple.clinic.facility.Facility
 import org.simple.clinic.facility.change.confirm.di.ConfirmFacilityChangeComponent
 import org.simple.clinic.widgets.BottomSheetActivity
 
 class ConfirmFacilityChangeSheet : BottomSheetActivity() {
 
   companion object {
     lateinit var component: ConfirmFacilityChangeComponent
+
+    private const val SELECTED_FACILITY = "seleceted_facility"
+
+    fun intent(
+        context: Context,
+        facility: Facility
+    ): Intent {
+      val intent = Intent(context, ConfirmFacilityChangeSheet::class.java)
+      intent.putExtra(SELECTED_FACILITY, facility)
+      return intent
```

Comment:
Should we inline the return, wdyt?
```suggestion
      return Intent(context, ConfirmFacilityChangeSheet::class.java)
          .putExtra(SELECTED_FACILITY, facility)
```

Reply:
I can. But I find this to be more readable. Plus we are following this everywhere else. So unless this is something that you feel will be useful, I'd rather not change it.